app/v1/models/errors/desErrors.scala

```diff
@@ -26,12 +26,12 @@ object DesErrorCode {
   implicit val reads: Reads[DesErrorCode] = Json.reads[DesErrorCode]
```

Comment:
Curious as to why we've kept the "DesErrorCode" stuff on lines 21-26?

Reply:
Fixed